packages/cxl-ui/scss/cxl-jw-player/cxl-jw-player-next-up.scss

```diff
@@ -1,3 +1,9 @@
-.jw-next-up-cta {
-  pointer-events: all;
+cxl-jw-player {
+  .jw-nextup {
+    display: none;
+  }
+
+  .jw-next-up-cta {
+    pointer-events: all;
+  }
 }
```

Comment:
But this should be a global style, if not using `::slotted()`?

Reply:
It's a peculiar case. Ideally everything would exists in the shadow dom, though JW Player doesn't like it. The player itself is in the light dom while everything else is in the shadow dom.

In this case you are correct. I have moved these styles.